designer/submodules/packages/components/src/components/usages/componentView.tsx

```diff
@@ -67,11 +76,32 @@ function Component(): JSX.Element {
         [statusFilterLabels, t],
     );
 
+    const theme = useTheme();
+    const getColor = useCallback(
+        (type, value) => {
+            switch (type) {
+                case "USAGE_TYPE":
+                    switch (value) {
+                        case UsagesFiltersUsageType.INDIRECT:
+                            return theme.palette.secondary.main;
+                        case UsagesFiltersUsageType.STRAIGHT:
+                            return theme.palette.primary.main;
+                    }
+            }
+            return getRandomColorByType(type);
```

Comment:
Can you write a comment why here is picked a random color? I know that it was done this way before your change, but now it is more explicit and I'm wondering why this is done that way

Reply:
its not random, based on string provided